spec/associations/active_record_extensions_spec.rb

```diff
@@ -234,6 +234,33 @@ def define_doctor_classes
 
           expect(patient.physicians).to contain_exactly(physician1, physician2)
         end
+
+        describe "with the :source option" do
+          it "finds ActiveHash records through the join model" do
+            Patient.has_many :doctors, through: :appointments, source: :physician
```

Comment:
@kbrock what do you think of this approach where we simply add the differently named associations and dont bother removing the "default" ones. I was kind of wanting to not have both associations there in the spec contexts, but it's not quite necessary and doesn't really effect the functionality.

One other thought is that I could take the approach here but also explicitly remove the associations and added methods, something like:
```rb
Appointment._reflections.delete('physician')
Appointment.class_eval do
  undef_method :physician
  undef_method :physician=
end
Appointment.belongs_to :doctor, class_name: 'Physician', foreign_key: :physician_id
```
... but that seems like a lot for something that really doesn't "do anything"; specs will pass with or without it.